src/h3lib/lib/vertex.c

```diff
@@ -0,0 +1,247 @@
+        {{9, 3}, {13, 0}, {18, 3}, {-1, 0}, {-1, 0}},   // base cell 104
+        {{12, 3}, {16, 1}, {17, 0}, {18, 5}, {-1, 0}},  // base cell 105
+        {{15, 1}, {16, 0}, {17, 5}, {-1, 0}, {-1, 0}},  // base cell 106
+        {{14, 3}, {9, 0}, {18, 1}, {13, 4}, {19, 0}},   // base cell 107 (pent)
+        {{15, 0}, {19, 1}, {-1, 0}, {-1, 0}, {-1, 0}},  // base cell 108
+        {{15, 0}, {16, 5}, {19, 1}, {-1, 0}, {-1, 0}},  // base cell 109
+        {{13, 3}, {18, 0}, {-1, 0}, {-1, 0}, {-1, 0}},  // base cell 110
+        {{13, 3}, {17, 1}, {18, 0}, {-1, 0}, {-1, 0}},  // base cell 111
+        {{14, 3}, {18, 1}, {19, 0}, {-1, 0}, {-1, 0}},  // base cell 112
+        {{16, 1}, {17, 0}, {18, 5}, {-1, 0}, {-1, 0}},  // base cell 113
+        {{14, 3}, {15, 5}, {18, 1}, {19, 0}, {-1, 0}},  // base cell 114
+        {{13, 3}, {18, 0}, {19, 5}, {-1, 0}, {-1, 0}},  // base cell 115
+        {{17, 1}, {18, 0}, {-1, 0}, {-1, 0}, {-1, 0}},  // base cell 116
+        {{15, 5}, {19, 0}, {17, 3}, {18, 2}, {16, 4}},  // base cell 117 (pent)
+        {{15, 5}, {18, 1}, {19, 0}, {-1, 0}, {-1, 0}},  // base cell 118
+        {{13, 3}, {17, 1}, {18, 0}, {19, 5}, {-1, 0}},  // base cell 119
+        {{18, 1}, {19, 0}, {-1, 0}, {-1, 0}, {-1, 0}},  // base cell 120
+        {{17, 1}, {18, 0}, {19, 5}, {-1, 0}, {-1, 0}},  // base cell 121
+};
+
+/**
+ * Get the number of CCW rotations of the cell's vertex numbers
+ * compared to the directional layout of its neighbors.
+ * @return Number of CCW rotations for the cell, or INVALID_ROTATIONS
+ *         if the cell's base cell was not found on this face
+ */
+int vertexRotations(H3Index cell) {
+    // Get the face and other info for the origin
+    FaceIJK fijk;
+    _h3ToFaceIjk(cell, &fijk);
+    int baseCell = H3_EXPORT(h3GetBaseCell)(cell);
+    int cellLeadingDigit = _h3LeadingNonZeroDigit(cell);
+
+    // Excluding the next line from branch coverage because we never reach
+    // the loop ending condition, we always return first
+    for (int i = 0; i < MAX_BASE_CELL_FACES; i++) {  // LCOV_EXCL_BR_LINE
+        BaseCellRotation rotation = baseCellVertexRotations[baseCell][i];
+        if (rotation.face == fijk.face) {
+            int ccwRot60 = rotation.ccwRot60;
+            // Check whether the cell crosses a deleted pentagon subsequence
+            if (_isBaseCellPentagon(baseCell)) {
+                if (cellLeadingDigit == JK_AXES_DIGIT &&
+                    fijk.face ==
+                        baseCellVertexRotations[baseCell][IK_AXES_DIGIT - 2]
+                            .face) {
+                    // Crosses from JK to IK: Rotate CW
+                    return (ccwRot60 + 5) % 6;
+                }
+                if (cellLeadingDigit == IK_AXES_DIGIT &&
+                    fijk.face ==
+                        baseCellVertexRotations[baseCell][JK_AXES_DIGIT - 2]
+                            .face) {
+                    // Crosses from IK to J: Rotate CCW
```

Comment:
IK to JK?

Reply:
Yeah, will update... I think I thought this was `J` at some point, though not sure why, since `IK` and `JK` are the neighboring directions across the deleted subsequence.